.travis.yml

```diff
@@ -1,6 +1,5 @@
-language: generic
-dist: xenial
-sudo: required
```

Comment:
This is not quite needed.

Reply:
why not? switching from xenial to bionic may have lots of implications about what can be compiled.

Reply:
That might be true, but the `sudo: required` is not needed though.

Reply:
They changed in the last Nov:
https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

Reply:
i'm fine removing the sudo key, only.